helmfile/charts/grafana-ops/templates/dashboard-configmap .yaml

```diff
@@ -0,0 +1,40 @@
+apiVersion: v1
```

Comment:
There seems to be a space in the name of this file.

Reply:
Good catch, fixed!